django-vue-graphql-update/source_code_step_3/back_end/backend/settings.py

```diff
@@ -0,0 +1,131 @@
+"""
+Django settings for backend project.
+
+Generated by 'django-admin startproject' using Django 5.0.2.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/5.0/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/5.0/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/5.0/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = (
+    "django-insecure-nv&xe8r)*pw1b0n^n#hax!2i%k-!*ek(0h)2td))j-(5eomvda"
+)
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
+
+ALLOWED_HOSTS = []
+
+
+# Application definition
+
+INSTALLED_APPS = [
+    "blog.apps.BlogConfig",
```

Comment:
You moved this to the end of the list in the tutorial, please also apply that throughout the materials steps 🥵